contracts/contracts/lib/AssetHelper.sol

```diff
@@ -37,6 +37,10 @@ library AssetHelper {
         require(asset.kind == kind, "Unexpected asset");
     }
 
+    function equals(Asset memory asset, Asset memory asset2) internal pure returns (bool) {
+        return asset.tokenAddress == asset2.tokenAddress;
```

Comment:
Let's compare kinds too to make this code more defensive against future changes, e.g. we may introduce a third asset class identified only by kind, or by attributes other than tokenAddress.